www/src/index.css

```diff
@@ -0,0 +1,14 @@
+* ::-webkit-scrollbar-track {
+    border-radius: 10px;
+    background-color: #21242C;
+}
+
+* ::-webkit-scrollbar {
+    width: 6px;
+    background-color: #21242C;
+}
+
+* ::-webkit-scrollbar-thumb {
+    border-radius: 6px;
+    background-color: #2A2E37;
+}
```

Comment:
You can use the Sylesheet key in the theme.tsx of the design-system to do this. It does the same, just to answer your question.